### ISSUE TITLE
Productive Bees and BeeSourceful Issue?

### ISSUE THREAD

**Issue by mouse0270** [JDKDigital/productive-bees#4]
Happened to find two diamond bees and caught them. Afterword's I built a small area and incased the bees in a area. However shortly after the bees return to the hive my server crashes with the following error. I disabled BeeSourceful (really only had it because I like the idea of getting resource combs that have to be converted into a resource over getting just the resource).

I kinda hope you add that to your mod, as having both BeeSourceful and Productive Bee's seems kinda weird anyway since it already duplicates a lot of features from each other.

```
[m[1;31m[10:56:50] [Server thread/ERROR] [minecraft/MinecraftServer]: Encountered an unexpected exception
net.minecraft.crash.ReportedException: Ticking block entity
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:853) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:330) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:784) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:637) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_251] {}
Caused by: java.lang.NoSuchMethodError: net.minecraft.item.Item.func_200296_o()Lnet/minecraft/util/text/ITextComponent;
	at cy.jdkdigital.productivebees.world.storage.loot.functions.ConvertToComb.func_215859_a(ConvertToComb.java:26) ~[?:1.15.2-0.1.2] {re:classloading}
	at net.minecraft.world.storage.loot.LootFunction.apply(SourceFile:32) ~[?:?] {re:classloading}
	at net.minecraft.world.storage.loot.LootFunction.apply(SourceFile:21) ~[?:?] {re:classloading}
	at net.minecraft.world.storage.loot.functions.LootFunctionManager.func_216239_a(SourceFile:115) ~[?:?] {re:classloading}
	at net.minecraft.world.storage.loot.functions.ILootFunction.func_215857_a(SourceFile:16) ~[?:?] {re:classloading}
	at net.minecraft.world.storage.loot.ItemLootEntry.func_216154_a(SourceFile:28) ~[?:?] {re:classloading}
	at net.minecraft.world.storage.loot.StandaloneLootEntry$1.func_216188_a(SourceFile:60) ~[?:?] {re:classloading}
	at net.minecraft.world.storage.loot.LootPool.func_216095_b(LootPool.java:67) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.world.storage.loot.LootPool.func_216091_a(LootPool.java:90) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.world.storage.loot.LootTable.func_216114_a(LootTable.java:68) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.world.storage.loot.LootTable.func_216120_b(LootTable.java:80) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.world.storage.loot.LootTable.func_216113_a(LootTable.java:85) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at cy.jdkdigital.productivebees.tileentity.AdvancedBeehiveTileEntity.getBeeProduce(AdvancedBeehiveTileEntity.java:179) ~[?:1.15.2-0.1.2] {re:classloading}
	at cy.jdkdigital.productivebees.tileentity.AdvancedBeehiveTileEntity.lambda$tick$3(AdvancedBeehiveTileEntity.java:111) ~[?:1.15.2-0.1.2] {re:classloading}
	at net.minecraftforge.common.util.LazyOptional.ifPresent(LazyOptional.java:161) ~[?:?] {re:classloading}
	at cy.jdkdigital.productivebees.tileentity.AdvancedBeehiveTileEntity.func_73660_a(AdvancedBeehiveTileEntity.java:110) ~[?:1.15.2-0.1.2] {re:classloading}
	at net.minecraft.world.World.func_217391_K(World.java:473) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.world.server.ServerWorld.func_72835_b(ServerWorld.java:368) ~[?:?] {re:classloading,xf:fml:quark:change-sleeping-player-count,xf:fml:quark:add-rave-hook,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:849) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	... 4 more
[m[1;31m[10:56:50] [Server thread/ERROR] [minecraft/MinecraftServer]: This crash report has been saved to: C:\MC Server Soft\servers\Forge 31.1.49\.\crash-reports\crash-2020-05-08_10.56.50-server.txt
[m[32m[10:56:50] [Server thread/INFO] [minecraft/MinecraftServer]: Stopping server
```

**Comment by JaisDK** [comment]
Please update to 0.1.3 as the class throwing the error has been removed in the newest version.
I chose to integrate with BeeSourceful because I don't like to re-implement something that exists in another mod. The BeeSourceful centrifuge does come with some limitations and problems so I might do it in the end.
The bees from his mod does not integrate with my hive either, but you can disable the spawning of those nests in the config.

**Comment by mouse0270** [comment]
I forget your mod doesn't show up in the twitch client for mods and that I have to manually check for updates. I just updated this mod I will check for bugs in a bit.

We currently have all of the BeeSourceful hives set to false so they do not generate in the world, we had to do this because of the bug where Bee's wouldn't use hives from other mods and this was annoying. So we just stopped BeeSourceful from generate bees and hives.
